guides/user/pipeline/triggers/gcs/index.md

```diff
@@ -120,15 +120,15 @@ run our pipeline.
 
 Next, we must configure the trigger: 
 
-* __Type__ is "Pub/Sub".
-* __Pub/Sub System Type__ is "Google".
-* __Subscription Name__ depends on what you've configured in your Pub/Sub
-  configuration using Halyard.
-* __Attribute Constraints__ must be configured to include the pair `eventType`:
+* __Type:__ Pub/Sub
```

Comment:
This is a good change, but I would keep the colons outside of the boldface:

* __Pub/Sub System Type__: Google

Reply:
```suggestion
| Field | Value |
|-------|-------|
| __Type__ | Pub/Sub |
| __Pub/Sub System Type__ | Google |
| __Subscription Name__ | Depends on what you've configured in your Pub/Sub |
| __Attribute Constraints__ | Must be configured to include the pair `eventType`:`OBJECT_FINALIZE` (see the [docs (https://cloud.google.com/storage/docs/pubsub-notifications#events) for an explanation). 
| __Expected Artifacts__ | Must reference the artifact defined previously. |
```

Reply:
It _looks_ like you deleted the line
`* __Type:__ Pub/Sub`
...rather than swapping the `:` and the `__`.